index.html

```diff
@@ -100,7 +100,7 @@ <h1 class="page-header">Web Development Teaching Materials</h1>
            <td>2-3hrs
            <td> <a href="https://github.com/gdisf/teaching-materials/blob/master/ajax/README.md">Meetup description</a>
         <tr>
-           <td><a href="http://gdisf-js-apis.appspot.com/">JS303: Client-side APIs</a>
+           <td><a href="http://teaching-materials.org/">JS303: Client-side APIs</a>
```

Comment:
yes, that's correct. this should be  a relative link like the ones above to `/apis/`